src/api/app/controllers/webui/users/notifications_controller.rb

```diff
@@ -75,6 +75,8 @@ def set_selected_filter
 
   def show_more(notifications)
     total = notifications.size
+    total = Kaminari.config.default_per_page unless total.positive?
+
     flash.now[:info] = "You have too many notifications. Displaying a maximum of #{Notification::MAX_PER_PAGE} notifications per page." if total > Notification::MAX_PER_PAGE
     notifications.page(params[:page]).per([total, Notification::MAX_PER_PAGE].min)
```

Comment:
```suggestion
    per_page = total.positive? ? [total, Notification::MAX_PER_PAGE].min : Kaminari.config.default_per_page

    flash.now[:info] = "You have too many notifications. Displaying a maximum of #{Notification::MAX_PER_PAGE} notifications per page." if total > Notification::MAX_PER_PAGE
    notifications.page(params[:page]).per(per_page)
```
I'd prefer not to modify the `total` variable, which would make it lose its original meaning. Instead, we can create a `per_page` variable with the final value passed to the `.per` scope. This way, I see an improvement in the readability of the code.

What do you think?

Reply:
`total` is a local variable, I don't see any issue in overwriting it... And I don't really see an improvement by having two variables in such a small code. But I don't want to ping pong over such a small fix, so I'll accept the change.